book/player/intro.md

```diff
@@ -14,5 +14,12 @@ The applicaster player is a full fledged player implementing all of the optional
 * [iOS](/player/iOS.md)
 * [Android](/player/Android.md)
 * [TV Quick-Brick based platforms](/player/quickbrick-tv-player.md)
-* [Supporting Video Ads in Datasources](/player/ads-in-datasource.md)
-* [Player Analytics](/player/player-analytics.md)
\ No newline at end of file
+* [Player Analytics](/player/player-analytics.md)
+
+
+#### Datasource Protocols
+
+The following pages contain protocols for datasource providers to send additional data on a video item. 
```

Comment:
```suggestion
The following pages contain protocols for datasource providers to send additional data about a video item. 
```

Would it always be accurate to say "via the feed"?

Reply:
Yes, since a DSP communicates with the app only via a feed.. 
As for this comment - I was actually meant "on top of a video item", in a meaning of the extensions section of the item.. and not "about" the item.. 
How would you phrase that? "on top of a video item"?

Reply:
Changed it again. Let me know what you think about the new suggestion.